src/Microsoft.Framework.CommandLineUtils/CommandLine/AnsiConsole.cs

```diff
@@ -9,22 +9,34 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer)
+        private AnsiConsole(TextWriter writer, bool useAnsiCodes)
         {
             Writer = writer;
-            OriginalForegroundColor = Console.ForegroundColor;
+
+            _useAnsiCodes = useAnsiCodes;
+            if (_useAnsiCodes)
+            {
+                OriginalForegroundColor = Console.ForegroundColor;
+            }
         }
 
         private int _boldRecursion;
+        private bool _useAnsiCodes;
 
-        public static AnsiConsole Output = new AnsiConsole(Console.Out);
+        public static AnsiConsole GetOutput(bool useAnsiCodes)
```

Comment:
This changes the behaviour of this does it not? Previously everyone who called this would've gotten the same instance. Now they will all get a different one. Not sure it matters, but it is a thing. @davidfowl?